app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinViewModel.kt

```diff
@@ -8,21 +8,21 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileResetPinActivity]. */
 @ActivityScope
 class ProfileResetPinViewModel @Inject constructor() : ObservableViewModel() {
-  /** Variable for storing error message in first text input layout. */
+  /** The error message is shown if the second pin not matches with the first pin. */
   val pinErrorMsg = ObservableField("")
 
-  /** Variable for storing error message in second text input layout. */
+  /** The error message is confirmed with this pin. */
   val confirmErrorMsg = ObservableField("")
 
-  /** For confirming if the profile is admin or not. */
+  /** If the user is admin then he should have admin rights. */
   val isAdmin = ObservableField(false)
 
-  /** For storing input pin entered by user in first text input layout. */
+  /** This is used to store the user pin. */
```

Comment:
Can be simplified: 'The new PIN inputted by the user.'

Ditto for the confirmed, except it should be something like: 'The second PIN inputted by the user (is expected to be the same as [inputPin] to confirm the user knows this new number).'

Reply:
done